python/lsst/pipe/base/executionButlerBuilder.py

```diff
@@ -233,8 +237,10 @@ def buildLightweightButler(butler: Butler,
         Raise if something exists in the filesystem at the specified output
         location and clobber is `False`
     """
+    outputLocation = ButlerURI(outputLocation)
+
     # Do this first to Fail Fast if the output exists
-    if (dirExists := os.path.exists(outputLocation)) and not clobber:
+    if (dirExists := outputLocation.exists()) and not clobber:
```

Comment:
You might want to add a check here as well for `outputLocation.isdir()` so you can stop early if they've given you a file URI.